### DISCUSSION TITLE
[Projects Beta] Not able to filter by current day

### DISCUSSION THREAD

**Discussion by admin-goosebyte** [community/community#9675] [Projects and Issues]
When I have a date field in Project Beta I only get to pick from absolute values. I would like any issue with a deadline of the current day to show in my priority filter.

![image](https://user-images.githubusercontent.com/91681053/148283379-34e33c77-fe63-4105-85fc-827d97c133d8.png)

I saw some other discussions asking for syntax to filter by date range but those are still based on using absolute values.

I've been using the `iteration=@current` filter to show the work that needs to be done inside of a time frame. Inside of the UI you can see that 'Today' is identified in the same way as 'Current', so, **I propose the solution `deadline=@today`**. 

![image](https://user-images.githubusercontent.com/91681053/148289844-23c0dffd-cc41-42f6-9d4b-4f7b6f75f5ba.png)

![image](https://user-images.githubusercontent.com/91681053/148289779-eb871d6f-2464-4c75-b799-cd5edfebb078.png)

**Comment by alantgpl** [comment]
Seems to be possible now in the UI, eg. `deadline:">=@today" `
